mathext/gamma_inc.go

```diff
@@ -22,7 +22,7 @@ func GammaIncReg(a, x float64) float64 {
 
 // GammaIncRegComp computes the complemented regularized incomplete Gamma integral.
 //  GammaIncRegComp(a,x) = 1 - GammaIncReg(a,x)
-//                    = (1/ Γ(a)) \int_0^\infty e^{-t} t^{a-1} dt
+//                    = (1/ Γ(a)) \int_x^\infty e^{-t} t^{a-1} dt
```

Comment:
Can you add some spaced to make the equal signs line up (I imagine this was forgotten when the `Reg` was added.

Reply:
Done.